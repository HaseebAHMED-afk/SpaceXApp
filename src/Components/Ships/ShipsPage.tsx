import React from 'react'
import Ships from './Ships'

const ShipsPage = () => {
    return (
        <div>
            <Ships />
        </div>
    )
}

export default ShipsPage
