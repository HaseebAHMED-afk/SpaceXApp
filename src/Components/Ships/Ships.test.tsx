import React from 'react';
import {shallow} from 'enzyme';
import Ships from './Ships';

describe('Ship Test Case', () => {
    let container : any;

    beforeEach(() => container = shallow(<Ships />))
    it('renders the ships' , () => {
        expect(container).toMatchSnapshot();
    })
})
