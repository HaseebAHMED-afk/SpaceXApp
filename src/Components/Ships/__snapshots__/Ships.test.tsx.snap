// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ship Test Case renders the ships 1`] = `
<div
  className="ships"
>
  <div
    className="ship-deck"
  />
  <Footer />
</div>
`;
